runtime/scripts.vim

```diff
@@ -306,7 +306,9 @@ else
     set ft=virata
 
     " Strace
-  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve(' || s:line1 =~# '^__libc_start_main'
+  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve('
+    \ || s:line1 =~# '^__libc_start_main'
+    \ || s:line1 =~# 'execve(.*)\s\+= \d\+'
```

Comment:
I am wondering, why you need to change the regex. The first regex: `^\(\[pid \d\+\] \)\=[0-9:.]* *execve(` should already match the part you added  `execve(.*)\s\+= \d\+`. (because that part before the `execve` is already optional), so I wonder what exact problem you are trying to fix here.